### PR TITLE
Web and OpenAPI howto

### PR DESCRIPTION
**Pull request by slinkydeveloper**
I preferred to write the how to updated for vertx 3.7 (for the new error handling strategy), so the vertx version is of course 3.7 (I tested it with local snapshot installs)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+<1> Get the parsed parameters container
+<2> Extract the parsed path parameter
+<3> Search the pet
+<4> If pet is present, write the pet in the response
+<5> If pet is absent, fail the routing context with 404
+
+== Get the router
+
+Now we can generate the `Router` and add the "Not Found" and "Bad Request" error handlers:
+[source,java]
```
Comment (jponge) [inline README.adoc]: You should include code snippets using Asciidoc `include`, possibly with tags so you don't include the whole file but smaller bits.

**Review by jponge — CHANGES_REQUESTED** [review]
Great start, just make sure you use Asciidoc includes to make sure the docs remain sync'ed with the code.

**Comment by slinkydeveloper** [comment]
When I import all the verticle I cannot find any way to hide callouts. The only thing I find is: https://github.com/asciidoctor/asciidoctor/issues/2961 . Maybe we can add an `hide-callout` rule inside vertx howto theme and I apply when I import all the verticle?

**Comment by jponge** [comment]
Then just don't import all the verticle and show tag by tag :-)

You can always link to the full file.

**Comment by jponge** [comment]
@slinkydeveloper Could you add a _"See also"_ section? I think right now you don't have a how-to to cross-link with, but you can definitely link to the relevant pieces of documentation 😉

**Review by jponge — APPROVED** [review]
Just need some Gradle ;-)

**Review by jponge — APPROVED** [review]

**Comment by jponge** [comment]
@slinkydeveloper go for a merge?

**Comment by slinkydeveloper** [comment]
Can we wait for 3.7?

**Comment by jponge** [comment]
We can, but how-tos are still WIP so you could safely merge as well.

**Pull request merged**